Models/IonAuthModel.php

```diff
@@ -504,6 +504,11 @@ public function clearRememberCode(string $identity): bool
 			'remember_code'     => null,
 		];
 
+		// get identity of logged in user
+		if ($identity == $this->identityColumn) {
+			$builder = $this->db->table($this->tables['users']);
+			$identity = $builder->select($identity)->where('id', $this->session->get('user_id'))->get()->getRowArray();
```

Comment:
Why is this needed? We already have the identity and the user_id right?

Reply:
when logout $identity is equal to $this->identityColumn so it produces like this email => email therefore it can't update table users to clear remember code.  I add this line of code to produce like this email => example@mail.com so it can update table users, is that make sense?

Reply:
Could you do the update based on the user_id and remove one of the these queries?  Since the identify query is using the user_id anyway...

Reply:
If you want to remove this queries then we should remove calling clearRememberCode in logout because it pointless can't update users. Remember code in users table will be cleared when user logout and then login without check remember me

Reply:
is the remember me working after doing this changes? Please update

Reply:
You can see all changes in here : https://github.com/benedmunds/CodeIgniter-Ion-Auth/pull/1475/files

Reply:
After making this changes still i remember me functionality is not working